apiserver/facades/schema.json

```diff
@@ -5124,10 +5124,6 @@
                         }
                     }
                 },
-                "FinishRestore": {
```

Comment:
Shouldn't we still bump this version? Even if we delete the old implementations for a 3.0 release, it at least makes it clear that there is a new facade that no longer supports restore.

Reply:
Good idea